### ISSUE TITLE
Card images in card deck render with incorrect aspect ratio on IE11 when image type is JPG

### ISSUE THREAD

**Issue by reykjavikingur** [twbs/bootstrap#29396]
On Windows 10 using IE 11:

https://getbootstrap.com/docs/4.3/components/card/#card-decks

The HTML snippet for card decks shows `<img class="card-img-top">` being used for the card images in a card deck structure. However, the example actually shown uses `<svg class="card-img-top">`, which behaves differently on IE 11.

With an `svg` element in that place, as the width scales down, the aspect ratio is preserved but the image contents are cropped appropriately. With an `img` element, however, on most browsers, the aspect ratio is preserved, but on IE 11, it is not preserved.

The specific problems I would call out are:
1. Your HTML snippet is inconsistent with your rendered example of Card Decks in your documentation.
2. Your HTML snippet for card deck does not work appropriately on IE 11 when the image source is JPG.

**Comment by XhmikosR** [comment]
Could be we are missing `w-100 b-block`.

/CC @MartijnCuppens @ysds

**Comment by ysds** [comment]
This is a workaround:

```css
.card-deck {
  display: block;

  @media (min-width: 576px){
    display: flex;
  }
}

.card-img-top {
  flex: 0 0 auto;
}
```

https://jsbin.com/xuzapozeqa/1/edit?html,css,output

Maybe we should fix our CSS.

**Comment by XhmikosR** [comment]
If you think it's an issue with core then yes. If it's just a docs issue, I'd rather use the utils.

**Comment by MartijnCuppens** [comment]
This part:
```scss
.card-deck {
  display: block;

  @media (min-width: 576px){
    display: flex;
  }
}
```
Will be fixed by #28689.

We could just add the image fix?

**Comment by XhmikosR** [comment]
@MartijnCuppens Shouldn't we fix this in v4-dev too, though?
